### DISCUSSION TITLE
Query regarding length scale

### DISCUSSION THREAD

**Discussion by sushdash** [neperfepx/FEPX#47] [Q&A] [answered]
While running the simulations, one thing that I am always confused about is the length scale and unit! Can you please explain that a bit for me?

**Comment by mkasemer** [comment, marked as answer]
Units are discussed in the FEPX manual.

In summary: there is no inherent length scale in FEPX, as the base crystal plasticity models consider no lengths. Thus, you pick a length scale (for your domain, for instance, you can set as [mm]), and then use the appropriate units / magnitudes for all other values.
